zpay32/invoice_test.go

```diff
@@ -90,11 +93,22 @@ var (
 	// Must be initialized in init().
 	testPaymentHash     [32]byte
 	testDescriptionHash [32]byte
+
+	ltcTestNetParams chaincfg.Params
+	ltcMainNetParams chaincfg.Params
 )
 
 func init() {
 	copy(testPaymentHash[:], testPaymentHashSlice[:])
 	copy(testDescriptionHash[:], testDescriptionHashSlice[:])
+
+	// TODO(sangaman): create an interface for chaincfg.params
+	ltcTestNetParams = chaincfg.TestNet3Params
```

Comment:
Maybe a comment should be added to clarify what this block is doing? I was mildly confused by it.
Or maybe a better solution is to just pull the [applyLitecoinParams](https://github.com/lightningnetwork/lnd/blob/7abdd30a878efb3673ce4dfe0070570a6c9444c8/chainparams.go#L60) to this test file and use it, but it's ok either way.

Sorry if already discussed somewhere.

Reply:
Also, your squashed commit has all the messages merged, maybe modify it to be a single continuous paragraph instead.

Reply:
Agree that a comment explaining why/what this block is doing is a good idea 👍